### DISCUSSION TITLE
New issue label for maintenance?

### DISCUSSION THREAD

**Discussion by dcslagel** [kinverarity1/lasio#544] [General]
Hi, 

I recently created an issue: #543:  `cchardet alternatives for python 3.11+`.  I labeled it `bug` but it is really has to do with maintenance of Lasio software over time as versions of dependencies (or versions of Python) arrive or depart.   

Would it be okay to create a new label `maintenance` be added to issues and used for updates needed overtime for various changes in Python, Lasio dependencies and environment/infrastructure version changes (i.e. GitHub Actions)?

**Comment by kinverarity1** [comment]
Absolutely, sorry I missed this post. I'll add it now.

**Reply by dcslagel** [reply]
Thanks for adding.  I've started applying the new maintenance lable to relevant issues.
